build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -15,44 +15,16 @@
   </ItemGroup>
 
   <Target Name="Compile" DependsOnTargets="Prepare;
-                                           SetupStage;
-                                           CompileStage;
+                                           CompileCLI;
                                            PublishSdks;
                                            PublishTemplates;
                                            BuildProjectsForNuGetPackages;
                                            GetNuGetPackagesArchive;" />
 
-  <Target Name="SetupStage"
-          DependsOnTargets="Prepare">
-    <PropertyGroup>
-      <SrcDirectory>$(RepoRoot)/src</SrcDirectory>
-    </PropertyGroup>
-
-    <ItemGroup>
-      <CompileStageInputs Include="$(SrcDirectory)/**/*.cs" Exclude="$(SrcDirectory)/**/obj/**/*.cs" />
-      <CompileStageInputs Include="$(SrcDirectory)/**/*.csproj;$(SrcDirectory)/**/project.assets.json" />
-
-      <Stage Include="Stage1">
-        <Inputs>@(CompileStageInputs)</Inputs>
-        <StageDirectory>$(Stage1Directory)</StageDirectory>
-        <StageSymbolsDirectory>$(Stage1SymbolsDirectory)</StageSymbolsDirectory>
-        <DotnetDir>$(Stage0Directory)</DotnetDir>
-      </Stage>
-      <Stage Include="Stage2">
-        <Inputs>@(CompileStageInputs)</Inputs>
-        <StageDirectory>$(Stage2Directory)</StageDirectory>
-        <StageSymbolsDirectory>$(Stage2SymbolsDirectory)</StageSymbolsDirectory>
-        <DotnetDir>$(Stage0Directory)</DotnetDir>
-      </Stage>
-    </ItemGroup>
-  </Target>
-
-  <Target Name="CompileStage"
-          DependsOnTargets="SetupStage;
-                            InitCrossgenProps;"
-          Inputs="%(Stage.Inputs)"
```

Comment:
Are you sure it is safe to remove the incrementally check here? I wonder if this won't cause our signed dlls to be stomped over.